src/Package/Microsoft.Testing.Platform.DotNetTestClient/Microsoft.Testing.Platform.DotNetTestClient.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Could we drop the nuspec and do directly from project or is it hard for this setup?

Reply:
it should not be hard, I was just copying what you do in other projects

Reply:
removed